ans_task.yml

```diff
@@ -0,0 +1,109 @@
+---
+- name: Update packages and install Apache2
+  hosts: localhost
+  become: yes
+
+  tasks:
+    - name: Update package list
+      apt:
+        update_cache: yes
+
+    - name: Install Apache2
+      apt:
+        name: apache2
+        state: present
+
+- name: Create directory and index.html
+  hosts: localhost
```

Comment:
@shastish , since all of these tasks must be run on the server we are gonna run against, we dont need to seperate tasks as multiple plays

Reply:
Hi @saikumarch7548 I made the changes

Reply:
Please change in all the other places as well, for example line numbers 13,30,40,50,87 and 98